puppeteer-tests/src/comments/place-comment.spec.tsx

```diff
@@ -0,0 +1,42 @@
+import { setupBrowser, wait } from '../utils'
+import type { Browser } from 'puppeteer'
+
+const TIMEOUT = 120000
+
+describe('Comments test', () => {
+  it(
+    'can place a comment',
+    async () => {
+      const { page, browser } = await setupBrowser(
+        'http://localhost:8000/p/?fakeUser=alice&Commenting=true',
+        TIMEOUT,
+      )
+
+      const signInButton = await page.waitForSelector('div[data-testid="sign-in-button"]')
+      await signInButton!.click()
+      const commentModeButton = await page.waitForSelector(
+        'div[data-testid="canvas-toolbar-comment-mode-connected"]',
+      )
+      await commentModeButton!.click()
+      const canvasControlsContainer = await page.waitForSelector('#new-canvas-controls-container')
+      await canvasControlsContainer!.click({ offset: { x: 500, y: 500 } })
```

Comment:
What is this meant to be clicking? This part looks very brittle to me, as it's highly dependent on the default project, editor layout, and window size

Reply:
it's not meant to click anything, at this point the only goal is to trigger the comment composer popup. We can refine this in the future when we're writing separate tests for canvas and scene comments